test/remote-connector.test.js

```diff
@@ -87,3 +91,55 @@ describe('RemoteConnector', function() {
     });
   });
 });
+
+describe('Custom Path', function() {
+
+  var test = this;
+
+  before(function(done) {
+    
+    var db_config = {
+      connector: loopback.Memory,
+      name: 'db'
+    };
+    var db = loopback.createDataSource('db', db_config);
+
+    var ServerModel_config = {
+      dataSource: 'db',
+      http: {
+        path: '/custom'
+      }
+    };
+
+    var ServerModel = loopback.PersistedModel.extend('TestModel', {foo: 'string'}, ServerModel_config);
+    ServerModel.attachTo(db);
+
+    server = test.server = createAppWithRest();
+    server.model(ServerModel);
+    server.dataSource(db, db_config);
```

Comment:
This setup is a bit convoluted. I am proposing to simplify it a bit.

```js
var server = test.server = createAppWithRest();

var db_config = { /*...*/ };
server.dataSource('db', db_config);

var ServerModel = server.registry.createModel('TestModel', { foo: 'string' });

var ServerModel_config = { /*... */  };
server.model(ServerModel, ServerModel_config);
```

BTW I personally tend to inline `*_config` variables in the place where they are used, especially when they are used only once.

```js
server.dataSource('db', { connector: 'memory' });
```